sql_script/create_tables.sql

```diff
@@ -1,15 +1,15 @@
-CREATE TABLE IF NOT EXISTS user (
-    user_id VARCHAR(64) PRIMARY KEY,
-    user_type VARCHAR(32) NOT NULL,
+CREATE TABLE IF NOT EXISTS User (
```

Comment:
It would be nice to write comments for every column
ex: `kind VARCHAR(64) NOT NULL COMMENT 'comment here',`

Reply:
Done